subt/docker/robotika/run_solution.bash

```diff
@@ -34,6 +34,7 @@ while [ -z "$ROBOT_DESCRIPTION" ]; do
 done
 echo "Robot description is '$ROBOT_DESCRIPTION'"
 
+grep -q ssci_x2_sensor_config <<< $ROBOT_DESCRIPTION && IS_SSCI_X2=true || IS_SSCI_X2=false
 grep -q ssci_x4_sensor_config <<< $ROBOT_DESCRIPTION && IS_X4=true || IS_X4=false
```

Comment:
These two ignore the actual configuration - x4 config 1 does not have depth camera, x2 has only one configuration so it is ok for now - but would it be better to be rather defensive? Or is this intentional design decision?